packages/core/src/components/plekinfo-card/plekinfo-card.interfaces.ts

```diff
@@ -0,0 +1,7 @@
+export interface DsoPlekinfoCardClickEvent {
```

Comment:
Deze is nu geprefixed, er is nog één andere interface die we prefixen maar over het algemeen doen we dat niet:

![image](https://github.com/user-attachments/assets/1fd45cd7-8168-4c17-9ce1-9ec854985b9c)

Reply:
Gedaan